Ginger/GingerCoreNET/ActionsLib/ActionsConversion/eConversionStatus.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Amdocs.Ginger.CoreNET
+{
+    /// <summary>
+    /// This enum is used to hold the status for conversion
+    /// </summary>
+    public enum eConversionStatus
```

Comment:
no need seperate class for this enum, you can add in BusinessFlowToConvert.cs outside class

Reply:
removed